packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -159,8 +159,22 @@ const node = mesh.addVirtualNode('virtual-node', {
       unhealthyThreshold: 2,
     },
   })],
+  backendDefaultsClientPolicy: appmesh.ClientPolicy.fileTrust({
+    certificateChain: 'path-to-certificate',
+  }),
   accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout'),
 });
+
+const virtualService = new appmesh.VirtualService(stack, 'service-1', {
+  virtualServiceName: 'service1.domain.local',
+  mesh,
+  clientPolicy: appmesh.ClientPolicy.fileTrust({
+    certificateChain: 'path-to-certificate',
+    ports: [8080, 8081],
+  }),
+});
+
+node.addBackend(virtualService);
```

Comment:
This is a weird example, right? Because if I understand correctly, because the `VirtualService` defines its own `clientPolicy`, the `backendDefaultsClientPolicy` on `VirtualNode` will be ignored?

If my understanding is correct, I would re-structure the example to be less artificial.